src/pages/Widget/Snippet/Snippet.tsx

```diff
@@ -38,13 +39,16 @@ const widgetURLfn = (config: WidgetConfig) => {
       .filter((m) => !m.disabled)
       .map((m) => m.id)
       .join(","),
+    isTitleShown: config.isTitleShown ? "true" : "false",
+    title: config.title ?? "",
+    description: config.description ?? "",
   };
   return (
     window.location.origin +
     appRoutes.donate_widget +
     "/" +
     config.endowment.id +
     "?" +
-    new URLSearchParams(params).toString()
+    new URLSearchParams(cleanObject(params)).toString()
```

Comment:
```suggestion
    new URLSearchParams(params).toString()
```
dont include empty `""` param in the snippet